src/components/screens/wallet/overview/balanceInfo/index.js

```diff
@@ -29,15 +29,19 @@ const BalanceInfo = ({
             />
           </div>
           {
-            activeToken === tokenMap.LSK.key && isWalletRoute ? (
-              <LockedBalanceLink activeToken={activeToken} isWalletRoute={isWalletRoute} />
+            activeToken === tokenMap.LSK.key ? (
+              <LockedBalanceLink
+                activeToken={activeToken}
+                isWalletRoute={isWalletRoute}
+                account={account}
+              />
             ) : null
           }
         </DiscreetMode>
       </div>
       <ActionBar
         address={address}
-        username={username}
+        username={account?.dpos?.delegate?.username}
```

Comment:
I think using dpos is always returned on every account even though its not a delegate I think to reduce the number of uncertainty (i.e by using the optional changing `?.`) consider doing this

```suggestion
        username={account.dpos?.delegate?.username}
```